momentum_scanner_intraday.py

```diff
@@ -114,14 +108,11 @@ def get_high_returns(
     ]
     if df.empty:
         return df
-    # df = df[df["24H Return"] >= 0] # dropping this line in case all tokens have negative returns
     df = metrics.add_7drets(df)
-    df = add_intraday_rets(df, lag_return)
-    df[lag_col] = df[lag_col].apply(lambda x: round(x * 100, 2))
-    # df = df[df[lag_col] >= 0]
-    df = df.sort_values(by=lag_col, ascending=False)
```

Comment:
I would keep the sorting as it's not computationally costly but could be useful for certain use cases.

Reply:
I find these functions easier to understand if they don't contain lines which don't really do anything; I don't think there's a use case for this line and it doesn't change the final output.